pkg/etcdenvvar/envvarcontroller.go

```diff
@@ -57,6 +60,16 @@ func NewEnvVarController(
 	networkInformer configv1informers.NetworkInformer,
 	eventRecorder events.Recorder,
 ) *EnvVarController {
+
+	revGauge := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "etcd_operator",
+		Name:      "static_pod_current_revision",
```

Comment:
Checking one of [your presubmits here][2] in [PromeCIeus][1]:

```
sort_desc(count by (__name__) ({__name__=~".*revision.*"}))
```

finds:

```
etcd_debugging_auth_revision{}
etcd_debugging_mvcc_compact_revision{}
etcd_debugging_mvcc_current_revision{}
kube_statefulset_status_current_revision{}
kube_statefulset_status_update_revision{}
```

So... I don't see your new metric.  And maybe it should be called `static_pod_status_current_revision`?  Squatting on a generic metric name with etcd-specific code feels brittle...  `etcd_static_pod_status_current_revision`?  Or get the Gauge definition landed in library-go, where all interested static-pod consumers can share it?

[1]: https://promecieus.dptools.openshift.org/
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-etcd-operator/852/pull-ci-openshift-cluster-etcd-operator-master-e2e-agnostic/1535170318547554304